app/views/attachments/_attachment_box.html.erb

```diff
@@ -46,7 +46,7 @@
 
                 <button
                   class="btn btn-transparent js-attachment-url-copy"
-                  data-clipboard-text="!<%= project_node_attachment_path(node.project, node, attachment.filename) %>!"
+                  data-clipboard-text="!<%= "\n!" + project_node_attachment_path(node.project, node, attachment.filename) + "!\n" %>!"
```

Comment:
```suggestion
                  data-clipboard-text="<%= "\n!" + project_node_attachment_path(node.project, node, attachment.filename) + "!\n" %>"
```

Otherwise, the exclamation points `!` will double.`